client-presentation/src/features/concrete-features/storage.feature.js

```diff
@@ -0,0 +1,48 @@
+import Vue from 'vue';
+import {
+    Feature,
+    specificationType
+} from '../feature';
+import {
+    makeExampleId
+} from './features-utils';
+
+const exampleUsage =
+`// store a value in the persistent storage
+window.localStorage.myKey = 'value';
+
+// read a value from the storage
+console.log(window.localStorage.myKey); // prints 'value'
+
+// remove a value from the storage
+window.localStorage.removeItem('myKey');
+`;
+
+export default new Feature('storage',
+    exampleUsage,
+    () => ({
+        component: Vue.component(makeExampleId('storage'), {
+            data: {
+                value: window.localStorage.myKey || ''
+            },
+            template: `<div>
+                        <input type="text" placeholder="Data to store" v-model="value"/>
+                    </div>`,
+            watch: {
+                value: function (value) {
+                    window.localStorage.myKey = value;
+                }
+            }
+        }),
+        infoArray: [`Refresh the page and note the value is persisted.`]
+    }), {
+        test: () => window.localStorage,
+        specification: specificationType.STANDARD
+    }, {
+        test: () => window.sessionStorage,
+        specification: specificationType.STANDARD
+    }, {
+        test: () => window.indexedDB,
```

Comment:
I thought about adding `indexedDb` as separate feature - with example of loading image from it (offline). But it doesn't negates it here, legit to me.

Reply:
Makes sense. Again, I am biased with how I split things on whatwebcando, but definitely it would work separately, too.